js-exercises/duplicate-letters/duplicateLetters.js

```diff
@@ -0,0 +1,24 @@
+
+function duplicateLetters(args) {
+  let chars = {};
+  let maxduplicates = 0;
+  for (let i = 0; i < args.length; i++) {
+    if (chars.hasOwnProperty(args[i])) {
+      chars[args[i]] += 1;
+    } else {
+      chars[args[i]] = 1;
+    }
+
```

Comment:
Why is there a blank line here? Blank line are used to separate logical blocks of code.